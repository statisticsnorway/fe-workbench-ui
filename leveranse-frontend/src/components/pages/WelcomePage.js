import React from 'react';
import { Button, Grid, Image, Segment } from "semantic-ui-react";
import ssb_logo from '../../assets/SSB_logo.png';

let centerAlign = {
  textAlign: 'center'
}

class WelcomePage extends React.Component {
  handleClick = () => {
    this.props.history.push("/login");
  }

  render () {
    return (
      <div className="ui container">
        <Grid columns={1} centered>
          <Grid.Row verticalAlign='top'>
            <Grid.Column>
              <div className='ui container'>

              </div>
            </Grid.Column>
          </Grid.Row>
          <Grid.Row verticalAlign='top'>
            <Grid.Column>
              <div className='ui container'>
                <Image src={ssb_logo} size='large' centered/>
              </div>
            </Grid.Column>
          </Grid.Row>

        </Grid>
        <Segment textAlign='center' key='massive' size='massive' secondary>
          Velkommen til Arbeidsbenk
        </Segment>
        <div className='ui container' style={centerAlign}>
          <Button primary onClick={this.handleClick} size='huge'>Login</Button>
        </div>
      </div>
    )
  }
}

export default WelcomePage;