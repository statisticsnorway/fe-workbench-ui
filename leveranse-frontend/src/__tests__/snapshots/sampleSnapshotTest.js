import React from 'react';
import WelcomePage from '../../pages/WelcomePage';
import renderer from 'react-test-renderer';

it('test component using snapshot', () => {
  const tree = renderer
    .create(<WelcomePage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});