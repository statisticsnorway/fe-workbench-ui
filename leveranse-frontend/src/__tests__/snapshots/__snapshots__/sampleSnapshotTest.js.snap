// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component using snapshot 1`] = `
<div
  className="ui container"
>
  <div
    className="ui centered one column grid"
  >
    <div
      className="top aligned row"
    >
      <div
        className="column"
      >
        <div
          className="ui container"
        />
      </div>
    </div>
    <div
      className="top aligned row"
    >
      <div
        className="column"
      >
        <div
          className="ui container"
        >
          <img
            className="ui large centered image"
            src="ssb_logo.png"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="ui massive secondary center aligned segment"
  >
    Velkommen til Arbeidsbenk
  </div>
  <div
    className="ui container"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <button
      className="ui huge primary button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      Login
    </button>
  </div>
</div>
`;
